src/thriveopedia/Thriveopedia.cs

```diff
@@ -148,6 +153,12 @@ public override void _Ready()
 
         // Add all pages not associated with a game in progress
         AddPage("Museum");
+        AddPage("WikiRoot");
+
+        foreach (var page in ThriveopediaWikiPage.GenerateAllWikiPages())
```

Comment:
Would it be better to only generate the wiki pages when the thriveopedia is opened? I fear we'll have hundreds of wiki pages which would get initialized here even if the player doesn't open the thriveopedia in the editor or stage. Right now the stage and editor have separate options menu instances meaning that swapping between the editor and stage re-initializes the entire wiki each time.

Reply:
This would lead to a change in the order of pages, with the wiki coming after the current world details in the tree when in game, but I think that would be better actually. I'll make the change.

Reply:
I guess that makes sense. But it would also be possible for the "static" pages to also only be initialized when required if the order eventually turns into a concern.

Reply:
Ah my bad, the only page we need to add in order here is WikiRoot, since the actual wiki pages are added as its children wherever it is in the tree. I think I'll move it anyway since as I said it might be better to show the current world information first.